### ISSUE TITLE
Z-Wave JS to MQTT: Version 0.28.0 has issues with assocations (lifeline)

### ISSUE THREAD

**Issue by lucode** [hassio-addons/addon-zwavejs2mqtt#289]
# Problem/Motivation
I did update to the newest ZwaveJS2Mqtt version. And made a re-interview of all the nodes.
After that I figured out that my Fibaro Wally Double Switch FGWDSEU-221 doesn’t transfer the On/Off states from the node to HA anymore.
I did try to play around with associations, but couldn’t figure how to fix it.

```
zwavejs2mqtt: 5.10.1
zwave-js: 8.6.1

Z-Wave JS to MQTT: 0.28.0
```
It did also occur that  I couldn't change any associations, but can't figure out what triggered this. At the beginning it did work. 
Finally I did downgrade to the previous version of HA Z-Wave JS to MQTT and now everything works again.
```
zwavejs2mqtt: 5.8.0
zwave-js: 8.4.1

Z-Wave JS to MQTT: 0.27.0 
```

## Expected behavior

> Reporting of the on/off state of the Switch 
## Actual behavior

> No Reporting of the on/off state of the Switch 

## Steps to reproduce

> Run Z-Wave JS to MQTT: 0.27.0 with Fibaro Wally Double Switch FGWDSEU-221

Workaround downgrade:
 ```
zwavejs2mqtt: 5.8.0
zwave-js: 8.4.1

Z-Wave JS to MQTT: 0.27.0 
```

**Comment by lucode** [comment]
Another sensor that is affected with this bug is the Eurotronic Spirit see:
https://github.com/zwave-js/node-zwave-js/issues/3593

**Comment by AlCalzone** [comment]
This **might** already be fixed by https://github.com/zwave-js/node-zwave-js/pull/3654 which is available in node-zwave-js 8.7.3+

**Comment by lucode** [comment]
Ok seems to be fixed in the newest version.

```
Z-Wave JS to MQTT, V 0.30.0
zwavejs2mqtt: 5.12.0
zwave-js: 8.7.6
```

**Comment by TheDK** [comment]
Hi @lucode I started to experience the same, but I came from 0.27 and skipped everything updating directly to 0.31 - now for one of my Z Wave Networks most of the nodes do not report changes, but can be triggered / switched. If I do a refresh I get the most recent state.

I downgraded to 0.27 (even though 0.31 should not be affected), did a restart and a power cycle, with no effect. 

I checked group associations for a working node and not working node - both have lifeline to Node 1 (the controller) configured the same way.

Nothing happening in the logs when a node changes state by itself (all Fibaro moduels, Single Switches and Roller Shutter 3).

It appears as such the node wasn't sending anything or the controller wasn't picking it up.

I also restarted the node by cutting power, no effect.

Any idea where to go looking?

**Comment by lucode** [comment]
Did you reinterview the nodes? This is mandatory with such a big version change. And can you get the values when you refresh the values manually in Z-Wave JS to MQTT.

**Comment by TheDK** [comment]
> Did you reinterview the nodes? This is mandatory with such a big version change. And can you get the values when you refresh the values manually in Z-Wave JS to MQTT.

Manual refresh was working which is why I didn't think of a Re-Interview (as values seemed to be matching if manually triggered, but exactly that was the point. I didn't notice so far that neither the update itself nor a complete re-boot or power cycle triggers a full refresh - but the update dialog says it might be necessary, so this one is on me.

Long story short: Refresh solves the problem. Thanks for the input and your time!

I will no do the update again with a full refresh and expect it to be working fine. Will report back if that shouldn't be the case.

**Comment by TheDK** [comment]
This appears to be a little harder then just do "Re-Interview all Nodes" - here's what I noticed for my Z-Wave mesh with 19 nodes, all mains powered (no battery-powered nodes), one floor, very well meshed and rock solid for two years:

- After the update, before the re-interview, only 2-3 nodes out of 19 report back correct states.
- Re-Interview, some dropped messages - ~2/3 of the nodes report back correctly.
- Another Re-Interview  - again, most of the nodes work, but different ones.

So, given that even in what appear to be rather good conditions, it is very hard to get a completely successful re-interview done. That means going through all nodes one by one and trigger a manual re-interview for each node that doesn't work. That way I got it up and running again.

Just in case some with similar issues finds this...
